### PR TITLE
Add Cli Arguments Card in single run overview tab

### PR DESCRIPTION
**Pull request by KaroMourad**
- Add cli arguments card
- Fix react/display-name eslint config from error to off

### CODE REVIEW

File: aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTab.tsx [hunk 1]
```diff
@@ -4,15 +4,17 @@ import ErrorBoundary from 'components/ErrorBoundary/ErrorBoundary';
 
 import { ANALYTICS_EVENT_KEYS } from 'config/analytics/analyticsKeysMap';
 
-import * as analytics from 'services/analytics';
+import RunOverviewTabCLIArgumentsCard from 'pages/RunDetail/RunOverviewTab/components/CLIArgumentsCard/RunOverviewTabCLIArgumentsCard';
+import RunOverviewSidebar from 'pages/RunDetail/RunOverviewTab/components/RunOverviewSidebar/RunOverviewSidebar';
```
Comment (rubenaprikyan) [inline aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTab.tsx:8]: Why use alias?

File: aim/web/ui/.eslintrc [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
     "react/react-in-jsx-scope": "off",
+    "react/display-name": "off",
```
Comment (rubenaprikyan) [inline aim/web/ui/.eslintrc:32]: We are using display-name or our components in time of debugging.

**Review by rubenaprikyan — APPROVED** [review]
lgtm!

File: aim/web/ui/.eslintrc (hunk 2 above)
Comment (KaroMourad) [reply aim/web/ui/.eslintrc:32]: yes but when u want to create an arrow function which returns jsx it is impossible with this eslint config

File: aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTab.tsx (hunk 1 above)
Comment (KaroMourad) [reply aim/web/ui/src/pages/RunDetail/RunOverviewTab/RunOverviewTab.tsx]: fixed

**Review by arsengit — CHANGES_REQUESTED** [review]
Lets display '-' instead of empty cells
Screen Shot 2022-03-06 at 22 47 54

**Review by arsengit — APPROVED** [review]

**Pull request merged**